lib/events.js

```diff
@@ -52,11 +52,15 @@ Object.defineProperty(EventEmitter, 'defaultMaxListeners', {
   get: function() {
     return defaultMaxListeners;
   },
-  set: function(arg) {
+  set: function(n) {
     // force global console to be compiled.
     // see https://github.com/nodejs/node/issues/4467
     console;
-    defaultMaxListeners = arg;
+    // check whether the input is a positive number (which value is
+    // larger than zero and not a NaN).
```

Comment:
'which value is larger than zero' should probably instead be something like 'whose value is zero or greater'